### DISCUSSION TITLE
What does this error message mean: Can't connect to server?

### DISCUSSION THREAD

**Discussion by dfrancocci** [datenstrom/community#467] [Ask a question] [answered]
Hi. I'm enjoying using datenstrom yellow - I've set up a small blog, and installed some extensions. 

I tried to use the 'Updates' feature in the 'Settings' menu, but I am getting the message:

> **Server error**
> Something went wrong. Can't connect to server 'https://github.com'!

I'm guessing there may be something missing in the web server environment - do you have any suggestions, so that I can contact my web hosting provider?

In the system.ini file I can see the entry:
```
UpdateExtensionUrl: https://github.com/datenstrom/yellow-extensions
```

Here are the details of my set up:

Datenstrom Yellow 0.8.6, PHP 7.2.25, Apache Linux
Blog 0.8.4
Bundle 0.8.5
Command 0.8.6
Core 0.8.6
Edit 0.8.12
English 0.8.12
French 0.8.12
Gallery 0.8.2
German 0.8.12
Image 0.8.2
Markdown 0.8.8
Meta 0.8.8
Stockholm 0.8.5
Update 0.8.9
Youtube 0.8.2

**Comment by markseuffert** [comment, marked as answer]
Thanks for the detailed information.

The error usually means there is no Internet connection or that the Internet connections is blocked. If you have a web server with ssh access, you can check it yourself with `curl` at the command line. Otherwise contact your web server provider. Let us know what you find out.

**Comment by dfrancocci** [comment]
Thanks for your reply. 

I contacted my hosting provider, and they confirmed that outgoing connections were blocked as default. They've now switched them on, and the 'Updates' feature is working fine. 

I'll close this issue - thanks for you help.
